api/desecapi/models.py

```diff
@@ -897,20 +897,36 @@ def _act(self):
         self.domain.save(update_fields=['renewal_state', 'renewal_changed'])
 
 
-def captcha_default_content():
-    alphabet = (string.ascii_uppercase + string.digits).translate({ord(c): None for c in 'IO0'})
-    content = ''.join([secrets.choice(alphabet) for _ in range(5)])
+def captcha_default_content(kind: str) -> str:
+    if kind == Captcha.Kind.IMAGE:
+        alphabet = (string.ascii_uppercase + string.digits).translate({ord(c): None for c in 'IO0'})
+        length = 5
+    elif kind == Captcha.Kind.AUDIO:
+        alphabet = string.digits
+        length = 8
+    else:
+        raise ValueError(f'Unknown Captcha kind: {kind}')
+
+    content = ''.join([secrets.choice(alphabet) for _ in range(length)])
     metrics.get('desecapi_captcha_content_created').inc()
```

Comment:
Consider parameterizing by kind. You can do that by adding `['kind']` as the last argument [here](https://github.com/desec-io/desec-stack/blob/master/api/desecapi/metrics.py#L21) (see other examples there), and then changing this line to

```
metrics.get('desecapi_captcha_content_created').labels(kind).inc()
```

We also could do that before the `if` construction if we want to catch unknown kinds as well in monitoring. (Not sure if that's a good idea.)

Reply:
done. No need to catch unknowns, the serializer doesn't allow that anyways